app/views/users/_sign_in_message.html.haml

```diff
@@ -0,0 +1,6 @@
+- if @sign_up
+  Welcome to morph.io! You're
+  = link_to "now watching", watching_user_path(@user)
```

Comment:
I don't think there's any need to let people know about this at this stage. They've just logged into morph for the first time. I think it's unlikely that this message would make much sense at this stage.

Maybe another way users could be informed is that the very first time they receive an email alert there's an introductory paragraph that explains why they are receiving that email and what they can do to change the settings.